### ISSUE TITLE
Discover nodes is failing from node properties window on network graph

### ISSUE THREAD

**Issue by apella12** [zwave-js/zwave-js-ui#3632]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Z-Wave JS UI version

zwave-js-ui: 9.9.0.09b107a

### ZwaveJS version

zwave-js: 12.4.4

### Describe the bug

Discover nodes is failing from node properties window on network graph

### To Reproduce

Go to Network graph, pull up 
![node12 discover 2024-03-03 155852](https://github.com/zwave-js/zwave-js-ui/assets/54947543/0d6fd21f-40d2-45ff-a7d0-83817409a8f7)
popup for Node (Node 12 in my case), hit discover, get failure message.

### Expected behavior

No failure. 

### Additional context

Captured a log file.  It appears that a refresh is called first on all the nodes and then the discover on node 12 fails.  I did see earlier that refresh can only be run every 60 seconds, so maybe that is the cause?
[refresh node12.txt](https://github.com/zwave-js/zwave-js-ui/files/14474607/refresh.node12.txt)

**Comment by robertsLando** [comment]
>  I did see earlier that refresh can only be run every 60 seconds, so maybe that is the cause?

Seems that the call to discoverNodeNeighbors returns `false`, that means the discover went wrong for some reason. The other api you called was `refreshNeighbors` that's different as it get neighbors from all nodes, discover is different as is actually tries to discover the neighbors while the other gets them from the cache

**Comment by apella12** [comment]
Only clarification is that I didn't call `refreshNeighbors`. Checked this am.  I think that was called when I switched to the network graph tab.  After waiting for that to finish I tried the discover.  Cleaner output, but still an issue.
```
2024-03-04 09:02:15.696 DEBUG SOCKET: Event ZWAVE_API emitted to wIiE9anPnXhLypDkAAAZ
2024-03-04 09:02:15.701 INFO Z-WAVE: Calling api discoverNodeNeighbors with args: [ 12, [length]: 1 ]
2024-03-04 09:02:19.248 INFO Z-WAVE: Success zwave api call discoverNodeNeighbors false
```
Also tried same on my other system this AM (Have a basement and main floor) and it worked okay on the main floor, so it is something specific to the one system and not overall issue (both are at same version levels). Still could use ideas, but I'll try some other things locally.

**Comment by robertsLando** [comment]
@apella12 Could you attach driver debug logs please?

**Comment by apella12** [comment]
Better yet, I'm going to close this.  I deleted all the manually entered priority routes and return routes from all my nodes and the discover works again.

**Comment by robertsLando** [comment]
:+1

**Comment by cadwizzard** [comment]
I've just come across this as discover neighbors has suddenly stopped working for me.
I had 61 nodes and it always worked. A few weeks go i added one priority route to one node. This week i've added 4 more nodes and the discover neighbours now just spins like when refreshing and never completes. Tried on all the new nodes.
Power cycled the controller, restarted HA etc.
I dont have a log yet as I was researching similar issues. Now I wonder if there is an issue when routes are set, its creates a discovery issue.
I noticed that when i relocated one node, it did find a new (although less great) route back to the controller which shows in the map - but neighbours still shows none
